src/ApiPayloadConverter.js

```diff
@@ -95,53 +95,54 @@ class ApiPayloadConverter {
 
   /**
    * @description Converts the mail payload for the ZeptoMail.
-   * @param {Object} originalPayload The original payload (provider agnostic). originalPayload has two components 1. api and 2. payload. 
+   * @param {Object} originalPayload The original payload (provider agnostic). originalPayload has two components 1. api and 2. payload.
```

Comment:
JSDoc notation for this is:

```suggestion
   * @param {Object} originalPayload The original payload
   * @param {String} originalPayload.api The provider API version.
   * @param {String} originalPayload.payload The payload to convert to be compatible with the provider API.
```